c/src/ml-api-service-remote.c

```diff
@@ -82,9 +81,9 @@ _mlrs_get_node_type (const gchar * value)
   if (!value)
     return node_type;
 
-  if (g_ascii_strcasecmp (value, "remote_sender") == 0) {
+  if (g_ascii_strcasecmp (value, "sender") == 0) {
     node_type = NNS_EDGE_NODE_TYPE_PUB;
-  } else if (g_ascii_strcasecmp (value, "remote_receiver") == 0) {
+  } else if (g_ascii_strcasecmp (value, "receiver") == 0) {
```

Comment:
It seems that now we can only set pub/sub when setting node_type.
At the link below, it says that the pub/sub pattern is a type of obsever pattern and is unaware of each other's existence.
https://handhand.tistory.com/271

In the case of training offloading, I think it would be necessary for the receiver to send the trained model back to the sender.
In this situation, if we use the pub/sub structure, we must create a new service.
Since nnstreamer-edge does not have a reply function, it seems difficult to solve this problem in the current situation.

Although it is not a necessary part of this PR, it would be better to consider the following parts in future development.
```
typedef enum {
  NNS_EDGE_NODE_TYPE_QUERY_CLIENT = 0,
  NNS_EDGE_NODE_TYPE_QUERY_SERVER,
  NNS_EDGE_NODE_TYPE_PUB,
  NNS_EDGE_NODE_TYPE_SUB,

  NNS_EDGE_NODE_TYPE_UNKNOWN,
} nns_edge_node_type_e;
```

`nns_edge_node_type_e` has query_client and server. 
If we set node type with query_client as the sender and query_server as the receiver, we can send two-way data transmission and requests.

Reply:
As you mentioned, it is not possible to send a reply in the current pub/sub structure.
As suggested, the node type was changed to query server/client, so server can send the reply now.
I also added related tests, so please review them again.
Thanks for the suggestion.